server/test/unit/server/webservices/views/test_consumers.py

```diff
@@ -503,8 +504,8 @@ def test_get_consumer_bindings_by_repoid(self, mock_repo_qs, mock_factory, mock_
         mock_serial.serialize.return_value = serial_resp
 
         request = mock.MagicMock()
-        consumer_bindings = ConsumerBindingsView()
-        response = consumer_bindings.get(request, 'foo', 'some-repo')
+        consumer_binding = ConsumerBindingView()
```

Comment:
since you created a new View i'd suggest to add a new class of tests for it separately, i.e put it into TestConsumerBindingView()